webview-ui/src/index.css

```diff
@@ -1,15 +1,35 @@
 @layer theme, base, components, utilities;
+
+/*
+Import order matters for Tailwind CSS v4 compatibility
+
+External @import statements MUST come before Tailwind imports because:
+1. CSS spec requires ALL @import statements to precede other CSS rules (except @charset/@layer)
+2. Tailwind CSS v4 expands its imports into thousands of lines of generated utility CSS
+3. If external imports come after Tailwind imports, they end up after the generated CSS
+4. This causes PostCSS to throw: "@import must precede all other statements"
+
+The correct order is:
+- @layer declarations (empty)
+- External @import statements (fonts, icons, etc.)
+- Tailwind @import statements (theme, utilities)
+- @config directives
+- Regular CSS rules
+*/
+
+/* External imports must come first, before Tailwind imports */
+@import url("../../node_modules/@vscode/codicons/dist/codicon.css");
+@import url("../node_modules/@fontsource/azeret-mono/index.css");
+
+/* Tailwind imports come after external imports */
```

Comment:
I don't think this is true, and in tailwindcss v4 we don't even need to import these anymore: https://tailwindcss.com/blog/tailwindcss-v4#simplified-installation

I have a wip to set up v4 correctly.